public/pages/TopNQueries/TopNQueries.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
```

Comment:
This file shouldn't be included in this PR, you might need to do a rebase after the overview page got merged.

Reply:
This is a file that does redirecting to the three pages, so it should exist and then the changes for each page will merge.